source/utilities.tex

```diff
@@ -17527,7 +17527,7 @@
 \indexlibrary{\idxcode{rank}}%
 \tcode{template<class T>\br
  struct rank;}      &
- If \tcode{T} names an array type, an integer value representing
+ If \tcode{T} denotes an array type, an integer value representing
```

Comment:
I think for template parameters we just want "is"; "denotes" should be used for typedefs.

Reply:
Done (here and elsewhere).